CMakeLists.txt

```diff
@@ -27,14 +27,13 @@ if (ARCH_NAME MATCHES "^ARM64" OR ARCH_NAME MATCHES "AARCH64")
   set(ARCH_NAME "ARM64")
 elseif (ARCH_NAME MATCHES "^ARM")
   set(ARCH_NAME "ARM") # Normalize ARMHF -> ARM
-elseif (ARCH_NAME MATCHES "I386")
+elseif (ARCH_NAME MATCHES "I386" OR ARCH_NAME MATCHES "I686")
```

Comment:
Nit: You could use `elseif (ARCH_NAME MATCHES "I[3-6]86")` so it matches other potential x86 architectures.

Reply:
Good idea, I'll do that.